### PR TITLE
[Hexagon] Add test for depthwise conv2d schedule

### PR DESCRIPTION
**Pull request by farshidsp**
This PR adds depth wise conv2d tests on hexagon target. 
cc @mehrdadh  @csullivan @Lunderberg

### CODE REVIEW

**Comment by mehrdadh** [comment]
This PR could fail due to long hexagon CI. It will be fixed after this: https://github.com/apache/tvm/pull/11132

File: tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    in_channel, in_size, stride = tvm.testing.parameters(
+        (32, 112, 1),
+        (64, 112, 2),
+        (128, 56, 1),
+        (128, 56, 2),
+        (256, 28, 1),
+    )
+
+
+class TestDepthwiseConv2D_More(BaseDepthwiseConv2D):
```
Comment (mehrdadh) [inline tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py:285]: can you rename this to `TestDepthwiseConv2D`?

**Review by mehrdadh — CHANGES_REQUESTED** [review]

File: tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    layout = tvm.testing.parameter("NCHW", "NHWC")
+    use_scale_shift = tvm.testing.parameter(True, False, ids=["with_scale_shift", "no_scale_shift"])
+    apply_relu = tvm.testing.parameter(True, False, ids=["with_relu", "no_relu"])
+
+    (batch, in_channel, in_size, channel_multiplier, kernel, stride) = tvm.testing.parameters(
+        (1, 64, 32, 1, 3, 1),
+        (1, 128, 64, 2, 5, 2),
+    )
+    padding = tvm.testing.parameter("VALID")
+    dilation = tvm.testing.parameter(1)
```
[the remark this answers is not in the document]
Comment (farshidsp) [reply tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py:295]: Done. Thanks for the suggestion.

File: tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py (hunk 1 above)
Comment (farshidsp) [reply tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py]: Yes.

File: tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+
+class BaseDepthwiseConv2D:
+    """Provides the test_conv2d test function, to be used by other test classes.
+
+    Test parameter sets are split out into different classes for
+    readability (e.g. used for mobilenet), and for restrictions
+    (e.g. implemented only for llvm).
+    """
+
+    use_scale_shift = tvm.testing.parameter(False, ids=["no_scale_shift"])
```
[the remark this answers is not in the document]
Comment (farshidsp) [reply tests/python/contrib/test_hexagon/topi/test_depthwise_conv2d.py]: Moved these two line. Thanks

**Review by mehrdadh — APPROVED** [review]
LGTM! Thanks @farshidsp!

**Pull request merged**